.hpsf-gitlab-ci.yml

```diff
@@ -0,0 +1,9 @@
+test:
+  stage: test
+  tags: [nvidia-gh200]
+  image: masterleinad/kokkos-nvcc:12.6.1
```

Comment:
Not sure if we want to change to something more "official".

Reply:
Probably. Would also need a recipe how to build it.

Reply:
```
FROM nvidia/cuda:12.6.1-devel-ubuntu24.04

RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub

RUN apt-get update && apt-get install -y \
        bc \
        wget \
        python3 \ 
        cmake \
        && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

Reply:
What about we find a better solution later so we can continue with automating the GitLab integration?

Reply:
Sure.  Any particular reason why you wrote your own dockerfile rather than reuse `scripts/docker/Dockerfile.nvcc`?

Reply:
I wasn't able to run the `cmake` installed that way
> /usr/bin/bash: line 156: /opt/cmake/bin/cmake: cannot execute binary file: Exec format error

Also I needed to install `python3`, see https://gitlab.spack.io/masterleinad/kokkos/-/jobs/12702052 (which we could of course also add to Dockerfile.nvcc).

Reply:
There is an ADDITIONAL_PACKAGES argument